src/Pbr.cc

```diff
@@ -212,6 +219,13 @@ Errors PbrWorkflow::Load(sdf::ElementPtr _sdf)
   this->dataPtr->emissiveMap = _sdf->Get<std::string>("emissive_map",
       this->dataPtr->emissiveMap).first;
 
+  if (_sdf->HasElement("light_map"))
+  {
+    sdf::ElementPtr lightMapElem = _sdf->GetElement("light_map");
+    this->dataPtr->lightMap = lightMapElem->Get<std::string>();
+    this->dataPtr->lightMapUvSet = lightMapElem->Get<unsigned int>("uv_set");
```

Comment:
Since `uv_set` is optional, should this `Get` call pass in the default value?

It may be interesting to add a test case without `uv_set` too.

Reply:
updated to pass in default value here, and changed one of the test case to check a  `<light_map>` without a `uv_set` attribute. 45f5cb2